packages/notebook/src/widget.ts

```diff
@@ -206,7 +206,9 @@ export class StaticNotebook extends WindowedList {
           (options.notebookConfig?.windowingMode ??
             StaticNotebook.defaultNotebookConfig.windowingMode) === 'full'
       }),
-      layout: new NotebookWindowedLayout()
+      layout: new NotebookWindowedLayout(),
+      renderer: StaticNotebook.defaultRenderer,
+      scrollbar: true
```

Comment:
We will want this behind a user setting, it's set to `true` for now for testers to see and use.